<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="15383" started-at="2020-01-29T14:59:16Z" finished-at="2020-01-29T14:59:32Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="15383" started-at="2020-01-29T14:59:16Z" finished-at="2020-01-29T14:59:32Z">
      <class name="test_Cases.testCase_1">
        <test-method status="PASS" signature="Extend()[pri:0, instance:test_Cases.testCase_1@72d1ad2e]" name="Extend" is-config="true" duration-ms="114" started-at="2020-01-29T20:29:16Z" finished-at="2020-01-29T20:29:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Extend -->
        <test-method status="PASS" signature="BefTest()[pri:0, instance:test_Cases.testCase_1@72d1ad2e]" name="BefTest" is-config="true" duration-ms="1" started-at="2020-01-29T20:29:16Z" finished-at="2020-01-29T20:29:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BefTest -->
        <test-method status="PASS" signature="BeforeMethod()[pri:0, instance:test_Cases.testCase_1@72d1ad2e]" name="BeforeMethod" is-config="true" duration-ms="4885" started-at="2020-01-29T20:29:16Z" finished-at="2020-01-29T20:29:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeMethod -->
        <test-method status="FAIL" signature="Reverse()[pri:1, instance:test_Cases.testCase_1@72d1ad2e]" name="Reverse" duration-ms="19" started-at="2020-01-29T20:29:21Z" finished-at="2020-01-29T20:29:21Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<af[]gsd> but was:<af[dgd]gsd>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<af[]gsd> but was:<af[dgd]gsd>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at test_Cases.testCase_1.Reverse(testCase_1.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Reverse -->
        <test-method status="PASS" signature="flush(org.testng.ITestResult)[pri:0, instance:test_Cases.testCase_1@72d1ad2e]" name="flush" is-config="true" duration-ms="3304" started-at="2020-01-29T20:29:21Z" finished-at="2020-01-29T20:29:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Reverse status=FAILURE method=testCase_1.Reverse()[pri:1, instance:test_Cases.testCase_1@72d1ad2e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flush -->
        <test-method status="PASS" signature="BeforeMethod()[pri:0, instance:test_Cases.testCase_1@72d1ad2e]" name="BeforeMethod" is-config="true" duration-ms="4154" started-at="2020-01-29T20:29:24Z" finished-at="2020-01-29T20:29:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeMethod -->
        <test-method status="FAIL" signature="Reverse2()[pri:1, instance:test_Cases.testCase_1@72d1ad2e]" name="Reverse2" duration-ms="1" started-at="2020-01-29T20:29:29Z" finished-at="2020-01-29T20:29:29Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<af[]gsd> but was:<af[dgd]gsd>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<af[]gsd> but was:<af[dgd]gsd>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at test_Cases.testCase_1.Reverse2(testCase_1.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Reverse2 -->
        <test-method status="PASS" signature="flush(org.testng.ITestResult)[pri:0, instance:test_Cases.testCase_1@72d1ad2e]" name="flush" is-config="true" duration-ms="3001" started-at="2020-01-29T20:29:29Z" finished-at="2020-01-29T20:29:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Reverse2 status=FAILURE method=testCase_1.Reverse2()[pri:1, instance:test_Cases.testCase_1@72d1ad2e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flush -->
      </class> <!-- test_Cases.testCase_1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
